### ISSUE TITLE
S4: Can't turn on Screen Notifications w/out turning on TalkBack

### ISSUE THREAD

**Issue by mcadams06** [lkorth/screen-notifications#32]
I have a Sprint S4. Android Version 4.4.2 I'm trying to turn Screen Notifications on, and it says: Screen Notifications wants permission to: Monitor your actions- Allows TalkBack to know when you are using an app." Ok great, like usual. 

I hit ok, and a new window pops up that says "TalkBack and the options below cannot both be turned on. Turn on TalkBack? (Air Gesture, air view, assistant menu, etc). I hit cancel because I don't want TalkBack on, but then it won't turn on Screen Notifications?

How do I turn on Screen Notifications without turning on TalkBack? This only started happening today.
